core-services/image-mirroring/_config.yaml

```diff
@@ -649,25 +649,27 @@ supplementalCIImages:
     image: registry.ci.openshift.org/ci/telco-runner:latest
   ci/che-e2e:next:
     image: quay.io/eclipse/che-e2e:next
+  stackrox/automation-flavors:openshift-4-0.10.14:
+    image: quay.io/stackrox-io/ci:automation-flavors-openshift-4-0.10.14
 ignoredSources:
-- image: registry.fedoraproject.org/fedora:latest
-  reason: https://issues.redhat.com/browse/RFE-5363
-- image: registry.ci.openshift.org/ocp/builder:rhel-7-golang-1.13
-  reason: broken image
-- image: registry.ci.openshift.org/ocp/builder:rhel-golang-1.13
-  reason: broken image
-- image: registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.11
-  reason: broken image
-- image: registry.ci.openshift.org/openshift/release:rhel-7-golang-1.13
-  reason: broken image
-- image: registry.ci.openshift.org/openshift/release:rhel-golang-1.13
-  reason: broken image
+  - image: registry.fedoraproject.org/fedora:latest
```

Comment:
Can we keep the config as before and only add the new image?

Reply:
We're using `yq` to automate the change and this is how it writes the updated YAML. So this would only be a one time change for this initial edit.